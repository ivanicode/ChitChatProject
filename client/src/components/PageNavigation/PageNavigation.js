import React from 'react';
import { NavLink } from 'react-router-dom';

function PageNavigation() {
    return (
        <nav>
            <ul className="mainNavigation">
                <li><NavLink className="navLink" activeClassName="activNavLink" to="/">Home</NavLink></li>
                <li><NavLink className="navLink" activeClassName="activNavLink" to="/friends">Znajomi</NavLink></li>
                <li><NavLink className="navLink" activeClassName="activNavLink" to="/history">Historia</NavLink></li>
                <li><NavLink className="navLink" activeClassName="activNavLink" to="/profile/edit">Edytuj profil</NavLink></li>                
            </ul>
        </nav>
    );
}

export default PageNavigation;