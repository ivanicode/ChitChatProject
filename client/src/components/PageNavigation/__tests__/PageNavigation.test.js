import React from 'react';
import {shallow} from 'enzyme';
import {PageNavigation} from '../PageNavigation';


describe('PageNavigation component', () => {
    it('should match snapshot', () => {
        const wrapper = shallow(<PageNavigation />)
        expect(wrapper).toMatchSnapshot()
    })
})