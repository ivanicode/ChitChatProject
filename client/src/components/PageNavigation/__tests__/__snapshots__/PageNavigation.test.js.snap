// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNavigation component should match snapshot 1`] = `
<nav>
  <ul
    className="mainNavigation"
  >
    <li>
      <NavLink
        activeClassName="activNavLink"
        className="navLink"
        to="/home"
      >
        Home
      </NavLink>
    </li>
    <li>
      <NavLink
        activeClassName="activNavLink"
        className="navLink"
        to="/friends"
      >
        Znajomi
      </NavLink>
    </li>
    <li>
      <NavLink
        activeClassName="activNavLink"
        className="navLink"
        to="/history"
      >
        Historia
      </NavLink>
    </li>
    <li>
      <NavLink
        activeClassName="activNavLink"
        className="navLink"
        to="/profile/edit"
      >
        Edytuj profil
      </NavLink>
    </li>
  </ul>
</nav>
`;
